Parse/build.gradle

```diff
@@ -5,7 +5,7 @@ apply plugin: 'com.github.kt3k.coveralls'
 apply plugin: 'com.jfrog.bintray'
 
 group = 'com.parse'
-version = '1.16.8-SNAPSHOT'
+version = rootProject.ext.commonLibVersion
```

Comment:
keep this version?  i think we don't need to have the same version tied to the gcm/fcm plugins...

Reply:
Do you plan to distribute the plugins separately? having everything at the same version helps understanding issues  :)